.circleci/config.yml

```diff
@@ -1073,12 +1101,57 @@ workflows:
           name: be-tests-presto-ee
           requires:
             - be-tests-ee
-          e: presto
+          e: presto-186
           before-steps:
             - wait-for-port:
                 port: 8080
           driver: presto
 
+      - test-driver:
+          name: be-tests-presto-jdbc-ee
+          requires:
+            - be-tests-ee
+          e: presto-254
+          before-steps:
+            - wait-for-port:
+                port: 8443
+            - run:
+                name: Create temp cacerts file based on bundled JDK one
+                command: cp $JAVA_HOME/lib/security/cacerts /tmp/cacerts-with-presto-ssl.jks
+            - run:
+                name: Install openssl
+                command: apk add openssl
+            - run:
+                name: Capture Presto server self signed CA
+                command: |
+                  while [[ ! -s /tmp/presto-ssl-ca.pem ]];
+                    do echo "Waiting to capture SSL CA" \
+                      && openssl s_client -connect localhost:8443 2>/dev/null </dev/null | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' > /tmp/presto-ssl-ca.pem \
+                      && sleep 1; done
+            - run:
+                name: Convert Presto CA from PEM to DER
+                command: openssl x509 -outform der -in /tmp/presto-ssl-ca.pem -out /tmp/presto-ssl-ca.der
+            - run:
+                name: Import Presto CA into temp cacerts file
+                command: |
+                  keytool -noprompt -import -alias presto -keystore /tmp/cacerts-with-presto-ssl.jks \
+                   -storepass changeit -file /tmp/presto-ssl-ca.der -trustcacerts
+          after-steps:
+            - run:
+                name: Capture max memory usage
+                command: cat /sys/fs/cgroup/memory/memory.max_usage_in_bytes
+                when: always
+          driver: presto-jdbc
+          extra-env: >-
+            MB_PRESTO_JDBC_TEST_CATALOG=test_data
+            MB_PRESTO_JDBC_TEST_HOST=localhost
+            MB_PRESTO_JDBC_TEST_PORT=8443
+            MB_PRESTO_JDBC_TEST_SSL=true
+            MB_PRESTO_JDBC_TEST_USER=metabase
+            MB_PRESTO_JDBC_TEST_PASSWORD=metabase
+            MB_ENABLE_PRESTO_JDBC_DRIVER=true
+            MB_PRESTO_JDBC_TEST_ADDITIONAL_OPTIONS="SSLTrustStorePath=/tmp/cacerts-with-presto-ssl.jks&SSLTrustStorePassword=changeit"
```

Comment:
for the other drivers we just put the env vars in the executor themselves, and you don't need to add an `extra-env` parameter to do that. Guess it doesn't make a big difference tho. Doing the same thing 2 different ways doesn't do a lot to help the maintainability of the CircleCI config, which is already pretty hairy TBH